.pre-commit-config.yaml

```diff
@@ -266,7 +266,7 @@ repos:
           The word(s) should be in lowercase." && exec codespell "$@"' --
         language: python
         types: [text]
-        exclude: ^airflow/_vendor/|^CHANGELOG\.txt$|^airflow/www/static/css/material-icons\.css$|^images/.*$
+        exclude: ^airflow/_vendor/|^RELEASE_NOTES\.txt$|^airflow/www/static/css/material-icons\.css$|^images/.*$
```

Comment:
```suggestion
        exclude: ^airflow/_vendor/|^RELEASE_NOTES\.rst$|^airflow/www/static/css/material-icons\.css$|^images/.*$
```

Reply:
Ugh! strikes again! I think that's the last one thankfully.